src/domain/Modals/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+import { Props, Variables } from '../../../common'
+import { Button } from '../../Buttons/Button'
+import { Inline } from '../../Layouts/Inline'
+import { InlineAlign } from '../../Layouts/Inline/Inline'
+import { Margin } from '../../Spacers/Margin'
+import { Text } from '../../Typographies'
+import { StyledReactModal } from './style'
+
+interface IDialogProps {
+  isOpen: boolean
+  handleClose: (event: (MouseEvent | KeyboardEvent | React.MouseEvent<HTMLButtonElement>)) => void
+  title: string
+  onPrimaryActionClick: (event: (React.MouseEvent<HTMLElement, MouseEvent>)) => void
+  primaryActionLabel: string
+  onSecondaryActionClick?: (event: (React.MouseEvent<HTMLElement, MouseEvent>)) => void
+  secondaryActionLabel?: string
+  variant: DialogVariant
+  componentContext: string
+}
+
+enum DialogVariant {
```

Comment:
should it be neutral and alert?

Reply:
I think the names can change depending on the context of the component. Like one colour can have many names:
green = approved, confirmed, success
red = noncompliant, declined, error

Reply:
In this case you're either performing a positive (blue) action or a negative (red) action